app/plugins.js

```diff
@@ -409,4 +417,12 @@ exports.getDecoratedBrowserOptions = defaults => {
   return decorateObject(defaults, 'decorateBrowserOptions');
 };
 
+exports.decorateSessionClass = Session => {
+  return decorateClass(Session, 'decorateSessionClass');
+};
+
+exports.decorateSessionOptions = defaults => {
+  return decorateObject(defaults, 'decorateSessionOptions');
```

Comment:
non breaking change

`exports.decorateSessionOptions` should come before `exports.decorateSessionClass ` to follow code structure of  `decorateObject ` and `decorateClass `

Reply:
You're right, I'll fix this